### ISSUE TITLE
[Request] Configurable speedgraph duration

### ISSUE THREAD

**Issue by Phr33d0m** [jfurrow/flood#346]
Hello,

Currently the speed graph only shows the last 3-5 minutes. It would be nice if we could configure the amount of time to display, for example the last 60 minutes or the last 48 hours. 

Thanks!

**Comment by jfurrow** [comment]
I agree, I've been wanting to do this for a long time, but I keep finding other issues that are a higher priority. Flood actually already collects this data in its history databases for what it's worth...

**Comment by Phr33d0m** [comment]
@jfurrow I've been through the code a bit (looking for something else than this) and saw what you're saying, thus I thought of putting this request here and whenever there's time for it you can track it easier. No hurry, as you said it's a very low priority.

Anyhow, from what I've seen there are a couple of functions defined [here](https://github.com/jfurrow/flood/blob/db5b50b7694513e8ee3988fe283a01d27afd1746/server/models/history.js), and after that you statically choose one of them (the 5min one) [here](https://github.com/jfurrow/flood/blob/81c33c4e58fbf818a38e2b18c43f8515da1f92ed/client/javascript/stores/TransferDataStore.js#L25).

Seems quite simple to add a config param in ```config.js``` to choose between one of the already defined interval functions. I think that will cover 99% of this feature request. I'll have a look and see if I can submit a pull req in a couple of hours.
